CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.2.2
+
+* Updated several dependencies [#10][]
```

Comment:
If the person who requested it wants to chat, I'd love to. I don't like giving contradictory code review, that just sucks for the PR submitter.

We should probably document examples. https://github.com/yola/wiki/wiki/Releasing-packages makes no mention of this particular detail. Neither does http://keepachangelog.com/